File: libpianoview/src/main/java/com/chengtao/pianoview/listener/OnPianoListener.java
package com.chengtao.pianoview.listener;


import com.chengtao.pianoview.entity.Piano;

public interface OnPianoListener {

    void onPianoInitFinish();

    void onPianoClick(Piano.PianoKeyType type, int group,
                      int positionOfGroup);
}

File: libpianoview/src/main/java/com/chengtao/pianoview/listener/OnPianoListener.kt
package com.chengtao.pianoview.listener

import com.chengtao.pianoview.entity.Piano.PianoKeyType

interface OnPianoListener {
    fun onPianoInitFinish()
    fun onPianoClick(type: PianoKeyType, group: Int, index: Int)
}